cron/poll_pull_requests.py

```diff
@@ -94,6 +97,12 @@ def poll_pull_requests(api):
             if is_approved:
                 __log.info("PR %d status: will be approved", pr_num)
 
+                # Make a tweet
+                message_twitter = datetime.datetime.ctime(
+                    datetime.datetime.now()) +\
+                    " - PR {pr_num} status: will be approved".format(pr_num=pr_num)
```

Comment:
This will spam the twitter feed, as it will happen every 30 seconds for every PR on its way to being approved. Either only do it once for every PR, or eliminate it altogether.